nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -181,10 +181,10 @@ def extcap_interfaces():
         :return: string with wireshark-compatible information
         """
         res = []
-        res.append("extcap {version=1.0}{help=https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver}{display=nRF 802.15.4 sniffer}")
+        res.append("extcap {version=1.0}{help=https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver}{display=nRF Sniffer for 802.15.4}")
```

Comment:
Can we here fix the wrong URL? Also please update version to: 0.7.2, which we need to release.